### ISSUE TITLE
Cannot call method '_getPK' of undefined

### ISSUE THREAD

**Issue by dottodot** [balderdashy/sails#2198]
sails 0.10.5
sails-mongo 0.10.4

When I try and retrieve a single model I get an error of

[Error (E_UNKNOWN) Encountered an unexpected error] Details:  TypeError: Cannot call method '_getPK' of undefined

but getting all works OK but if I try and populate user no results are returned so not sure if that's related.

model
```
module.exports = {
  adapter: 'revision',
  attributes: {
    user: {
      model:'user'
    },
    document_values: 'json',
    collection_name: 'string',
    document_id: 'string',
    type: 'string'
  }
};
```

controller
```
find: function(req, res, next) {
		var id = req.param('id');
		if (id) {
			Revision.findOne(id).exec(function(err, revision) {
				if (revision === undefined) return res.notFound();
				if (err) return next(err);
				res.send(revision);
			});
		} else {
			Revision.find().sort('createdAt DESC').exec(function(err, revisions) {
				res.send(revisions);
			});
		}
	}
```

**Comment by CWyrtzen** [comment]
Is this resolved?

**Comment by sailsbot** [comment]
Thanks for posting, @dottodot.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
